### PR TITLE
update cn doc

### PR DESCRIPTION
**Pull request by tyrone-yu**

### CODE REVIEW

File: docs/locales/zh_CN/LC_MESSAGES/reference/lang/manual.po [hunk 1]
```diff
@@ -107,17 +107,28 @@ msgid ""
...
-#: ../../reference/lang/manual.rst:105
+#: ../../reference/lang/manual.rst:107
+msgid ""
+"SCQL support ``export_options`` with limitations: only support '\"' or ''"
+" for **enclosing_character**; **ESCAPED BY** is not supported. "
+"**OPTIONALLY** in ``export_options`` controls quoting of fields, if "
+"omitted all fields are enclosed by the **enclosing_character**, otherwise"
+" only string fields are enclosed. see `mysql load data`_"
+msgstr "SCQL 在一定限制下支持 ``export_options``: 目前只支持 '\"' 或者 '' 作为 **enclosing_character**；暂时不支持 **ESCAPED BY** 语法。 "
+"``export_options``中的**OPTIONALLY**用于控制字段的引用行为，如果省略则所有类型的字段都会使用**enclosing_character**括起来，否则只有string类型的字段被括起来。"
```
Comment (jingshi-ant) [inline docs/locales/zh_CN/LC_MESSAGES/reference/lang/manual.po:121]: please add space between english and chinese

File: docs/locales/zh_CN/LC_MESSAGES/reference/p2p-deploy-config.po [hunk 2]
```diff
@@ -296,100 +298,130 @@ msgid "Whether to reveal group_mark directly for group by"
...
+#: ../../reference/engine-config.rst:49 ../../reference/engine-config.rst:65
+#: ../../reference/engine-config.rst:79 ../../reference/engine-config.rst:105
+#: ../../reference/engine-config.rst:109
+#: ../../reference/p2p-deploy-config.rst:117
+msgid "true"
+msgstr "true"
+
+#: ../../reference/p2p-deploy-config.rst:117
+msgid "Whether persist session to storage to support cluster mode"
+msgstr "集群模式下是否将 session 持久化到存储中去"
```
[the remark this answers is not in the document]
Comment (tyrone-yu) [reply docs/locales/zh_CN/LC_MESSAGES/reference/p2p-deploy-config.po]: 已修改

File: docs/locales/zh_CN/LC_MESSAGES/reference/lang/manual.po (hunk 1 above)
Comment (tyrone-yu) [reply docs/locales/zh_CN/LC_MESSAGES/reference/lang/manual.po]: 已添加

**Review by jingshi-ant — APPROVED** [review]
LGTM

**Pull request merged**